### PR TITLE
plan: make some editorial changes to the comments

### PR DESCRIPTION
**Pull request by queenypingcap**
@shenli @coocood Please take a look. Thanks

### CODE REVIEW

**Comment by shenli** [comment]
@queenypingcap Please sign the CLA.

**Comment by shenli** [comment]
LGTM

**Comment by hanfei1991** [comment]
LGTM

File: plan/physical_plan_builder.go [hunk 1]
```diff
@@ -271,10 +271,10 @@ func isCoveringIndex(columns []*model.ColumnInfo, indexColumns []*model.IndexCol
...
 }
 
-// convert2PhysicalPlan implements LogicalPlan convert2PhysicalPlan interface.
-// If no available index matches the required property, the returned physicalPlanInfo
-// will be table scan and has cost of MaxInt64, but it's OK because parent will call
-// convert2PhysicalPlan again with empty required property, so the plan with the lowest
+// convert2PhysicalPlan implements the LogicalPlan convert2PhysicalPlan interface.
+// If there is no index that matches the required property, the returned physicalPlanInfo
+// will be table scan and has the cost of MaxInt64. But this can be ignored because the parent will call
+// convert2PhysicalPlan again with the empty required property, so the plan with the lowest
```
Comment (coocood) [inline plan/physical_plan_builder.go]: with an empty

File: plan/physical_plan_builder.go [hunk 2]
```diff
@@ -271,10 +271,10 @@ func isCoveringIndex(columns []*model.ColumnInfo, indexColumns []*model.IndexCol
 	return true
 }
 
-// convert2PhysicalPlan implements LogicalPlan convert2PhysicalPlan interface.
-// If no available index matches the required property, the returned physicalPlanInfo
-// will be table scan and has cost of MaxInt64, but it's OK because parent will call
-// convert2PhysicalPlan again with empty required property, so the plan with the lowest
+// convert2PhysicalPlan implements the LogicalPlan convert2PhysicalPlan interface.
```
Comment (coocood) [inline plan/physical_plan_builder.go]: LogicalPlan is an interface name, it's proper noun.

**Comment by coocood** [comment]
LGTM

**Pull request merged**